streamlitDemo/utility.py

```diff
@@ -237,6 +238,7 @@ def realtime_audio_file_STT(audio_file_path, labelFlag="hidden"):
             sound_chunk.export(audio_chunk_temp_file, format ="wav")
             
             # Slicing of the audio file is done. transcribe audio chunks
+            time.sleep(2)
```

Comment:
adding a delay here to make sure response from API is received after the audio is heard